src/cli.rs

```diff
@@ -1246,6 +1265,24 @@ fn list_channels(
 	println!("]");
 }
 
+fn invoice_status(inbound_payments: PaymentInfoStorage, invoice: Invoice) {
+	if invoice.is_expired() {
+		println!("expired");
```

Comment:
HTLCStatus has no expired status. And it will never be `Failed` in case of invoice expiration.
So I added "expired" as a return value. This might make it harder to understand possible return values for the user, but I didn't know how to document it.